mautic/mautic
# [UX] More contrast for texts

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Based on user feedback:

> More contrast: replace the mid gray to a darker gray

The user is suggesting an improvement in the visual contrast of the interface by replacing mid-tone grays with darker shades. This indicates that the current color scheme may not provide sufficient contrast, potentially affecting readability and visual distinction between elements.

The user’s broader need is for a visually accessible and aesthetically pleasing interface that facilitates easy reading and navigation. They are seeking an enhancement in the visual hierarchy that allows users to quickly and effortlessly distinguish between different UI elements.

To address this feedback, this PR adjust the color palette to use darker grays for body text that offer better contrast against other elements. I ensured it meets WCAG accessibility standards for contrast ratios.


![image](https://github.com/mautic/mautic/assets/116097999/d131c6e9-a119-49da-8ea4-93c111ce695d)

The contrast ratio was increased by about ~7 points compared to before.




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Look at basically anything, the text will be darker
